vagrant-apim-vagrant-box-automation/boxes/api-manager-analytics/provisioner/provisioner.sh

```diff
@@ -42,6 +44,11 @@ if [ ! -f ${SOFTWARE_DISTRIBUTIONS}/${JDK_ARCHIVE} ]; then
     exit 1
 fi
 
+if [ ! -f ${SOFTWARE_DISTRIBUTIONS}/${WUM_ARCHIVE} ]; then
```

Comment:
Are we making WUM mandatory? Wouldn't it be better to make it optional?

Reply:
I made WUM mandatory because, in the run-time the user credentials are checked to see whether the user has the authentication to get the latest WUM updated pack using "wum init". If user does not have authentication, boxes will be created using the GA pack.

Reply:
Thanks Vimukthi for the explanation! As I see that's a technical problem that we may need to solve differently. Why would the user need to provide the WUM distribution if it is not needed for his/her usage?

Reply:
IMO, having WUM support in the Vagrant box is no harm. Else we have to keep two boxes which has WUM support and which doesn't. At the run-time we can ask from the user whether WUM update is needed or not. if user choose not to have the WUM update, we could easily skip those parts(at the run-time). But having WUM adds extra 3.1MB for the vagrant box.

Reply:
@imesh I'll make WUM optional, in case a user need to build a Vagrant box without WUM support. I'll add WUM support for all the boxes that will be hosted in the server.